src/components/responsive-flyout/styles/ResponsiveFlyout.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../styles/variables';
+@import '../../../styles/constants/media-queries';
+
+.flyout-overlay {
+    .overlay {
+        .bdl-OverlayHeader {
+            display: none;
+        }
+    }
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .flyout-overlay.responsive-flyout {
+            // cancels out the tether styling without having to set enabled=false
+            transform: none !important;
```

Comment:
We almost always want to avoid using `!important`, since it hampers extensibility and customization. Is there no other option here?